tests/arch/arm/arm_irq_zero_latency_levels/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+common:
+  filter: (CONFIG_ARMV6_M_ARMV8_M_BASELINE or CONFIG_ARMV7_M_ARMV8_M_MAINLINE) and not CONFIG_SOC_FAMILY_NRF
```

Comment:
I would like other reviewers comment on the test not exercised on NRF SoCs....

Reply:
Alternatives could be
- to change the irq priorities in the nrf device tree files from 5 to 4 (lots of possible side effects).
- to add a devicetree overlay for all impacted boards to the test which set the irq priority to 4 (would result in a lot of overlay files)

Reply:
it's the problem with fixed interrupt set for gpio. It is legacy issue, if i recall correctly when all interrupt levels were[ changed to `NRF_DEFAULT_IRQ_PRIORITY`](https://github.com/zephyrproject-rtos/zephyr/pull/20894/files#), gpio was left with `5` because it had different value than others (all set to `1`) and no one knows the reason why it's like that.

edit: I made some digging and found the [commit](https://github.com/zephyrproject-rtos/zephyr/commit/f2fb64b0752ff5f942e1b6e16809f3e20806196c) where it was set to 5 and it seemed to be an arbitrary value.

I would align it to `NRF_DEFAULT_IRQ_PRIORITY`.

https://github.com/zephyrproject-rtos/zephyr/blob/88ca4fb5336151f11857f966bb8ed1e11bf1962d/dts/arm/nordic/nrf52840.dtsi#L157

Reply:
The reason behind it could be to have a lower priority for the gpio than default.
I am happy to change it to NRF_DEFAULT_IRQ_PRIORITY, but be aware that ist concerns
nrf52840.dtsi
nrf5340_cpuapp_peripherals_ns.dtsi
nrf52833.dtsi
nrf52811.dtsi
nrf52832.dtsi
nrf9160ns.dtsi
nrf5340_cpunet.dtsi
nrf5340_cpuapp.dtsi
nrf52820.dtsi
nrf52810.dtsi
nrf52805.dtsi
nrf9160.dtsi
There might exist a couple of applications which implicitly rely on the lower prio gpio interrupt.
Should I intergrate the testcase back to tests/arch/arm/arm_irq_advanced_features then?